File: front/src/app/models/question.ts
import { Category } from "./category";
import { Answer } from "./answer";

export class Question {

    questionCode: number;
    text: string;
    category: Category;
    answers: Answer[];
    questionsPerGames: any;

    constructor(questionCode: number, text: string, category: Category, answers: Answer[], questionsPerGames: any) {
        this.questionCode = questionCode;
        this.text = text;
        this.category = category;
        this.answers = answers;
        this.questionsPerGames = questionsPerGames;
    }
}

File: back/src/main/java/org/bliz/trivial/controller/model/Question.java
package org.bliz.trivial.controller.model;

import java.io.Serializable;
import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Question implements Serializable {

	private static final long serialVersionUID = 1L;

	private Integer questionCode;

	private String text;

	private List<Answer> answers;

	private BlizzardGame category;

	private List<QuestionsPerGame> questionsPerGames;

}